docs/trex_examples.md

```diff
@@ -3,9 +3,53 @@ title: Dashboard Extension Samples
 layout: docs
 ---
 
-To examine the sample source code and to see how Tableau dashboard extensions work, you can clone or download the [Extensions API](https://github.com/tableau/extensions-api) repository on GitHub and run the samples. The samples are in the `Samples` folder. There is also a step-by-step tutorial you can follow in the `Tutorial` folder. 
+The best way to learn how to build your own extensions is to look at the sample code. To examine the sample source files to see how Tableau dashboard extensions work, you can clone or download the [Extensions API](https://github.com/tableau/extensions-api) SDK on GitHub and run the samples. 
+- To download the Extensions API SDK, if you have not already done so, see [Get Started]({{ site.baseurl }}/doc/trex_getstarted.html).
+
+- You can browse the sample code for the dashboard extensions in the [Samples](https://github.com/tableau/extensions-api/tree/master/Samples?=target="_blank") and the [Tutorial](https://github.com/tableau/extensions-api/tree/master/Tutorial?=target="_blank") folders on GitHub. 
+
+- You can also check out the dashboard extensions from the community, see [Community Extensions]({{ site.baseurl }}/community/).
+
+
+---
+
+
+**In this section**
+
+* TOC
+{:toc}
```

Comment:
What's this supposed to be?

Reply:
Ah, that's the kramdown macro that adds a table of contents at the top of a page. It grabs all the level 1, 2, .. etc. headings that are on the page puts in links. You can configure it in the _config.yml file to only grab certain headings.   I learned about the macro from Jared.

Reply:
Only works when you have GitHub pages.